velox/functions/prestosql/RowFunction.cpp

```diff
@@ -28,13 +28,28 @@ class RowFunction : public exec::VectorFunction {
       exec::EvalCtx& context,
       VectorPtr& result) const override {
     auto argsCopy = args;
+    auto nullsBuffer = BufferPtr(nullptr);
+    auto nullCount = 0;
+
+    const auto minSize = std::reduce(
+        args.begin(), args.end(), rows.end(), [](auto minVal, auto& child) {
+          return std::min(child->size(), minVal);
+        });
+
+    if (minSize < rows.end()) {
```

Comment:
It doesn't seem to be a valid use case. I think we should raise an error if some child vectors do not cover all 'rows'.

Reply:
There are test cases where in the child vectors end up being 0 sized - I saw this when running a test..

Reply:
Ideally if this function is called with a Selectivity vector (lets call it `rows`) then the inputs should have been fetched with the same input `rows`. This means the inputs should be atleast `rows.end()` size. If they are not, then it should be a bug.
@kgpai can you us more about what those tests are doing and why they require different sized inputs?